src/main.rs

```diff
@@ -226,7 +226,8 @@ fn run() -> Result<()> {
             let theme = assets.get_theme(config.theme.unwrap_or("Default"))?;
 
             if app.matches.is_present("list-languages") {
-                let languages = assets.syntax_set.syntaxes();
+                let mut languages = assets.syntax_set.syntaxes().to_owned();
+                languages.sort_by(|a, b| a.name.to_uppercase().cmp(&b.name.to_uppercase()));
```

Comment:
Would `sort_by_key` work here?

Something like
``` rust
languages.sort_by_key(|s| s.name.to_uppercase())
```

Reply:
Yep. And which one do you prefer? How about `sort_by_cached_key`? See rust-lang/rust#48639.

Reply:
I don't think we have to worry about performance here, so I would prefer `sort_by_key`, since it is easier to read, in my opinion (`sort_by_cached_key` is also fine, if it's already available in our minimum-required rustc version).